core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -291,14 +279,14 @@ class ReplicaFetcherThread(name: String,
     }
 
     val fetchData = builder.build()
-    val requestBuilder = JFetchRequest.Builder.
+    val requestBuilder = FetchRequest.Builder.
       forReplica(fetchRequestVersion, replicaId, maxWait, minBytes, fetchData.toSend())
         .setMaxBytes(maxBytes)
         .toForget(fetchData.toForget)
     if (fetchMetadataSupported) {
       requestBuilder.metadata(fetchData.metadata())
     }
-    ResultWithPartitions(new FetchRequest(fetchData.sessionPartitions(), requestBuilder), partitionsWithError)
```

Comment:
Am I missing something or did we drop the usage of `sessionPartitions`? Was that not needed?

Reply:
Nope, you're right. I misunderstood the usage.